decidim-bulletin-board-app/config/initializers/cors.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+return if Rails.application.config.settings.cors_origin_allowed.blank?
+
+Rails.application.config.middleware.insert_before 0, Rack::Cors do
+  allow do
+    origins "*"
```

Comment:
Should this be replaced by the environment variable?

```suggestion
    origins Rails.application.config.settings.cors_origin_allowed
```

Reply:
Oh wow... I totally forgot haha. Good catch @agustibr !